packages/azure-http-specs/specs/azure/resource-manager/operation-templates/lro.tsp

```diff
@@ -0,0 +1,203 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/spector";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using Azure.Core;
+using Azure.ResourceManager;
+using TypeSpec.OpenAPI;
+using Spector;
+
+namespace Azure.ResourceManager.OperationTemplates;
+
+@resource("lroResources")
+model LroResource is TrackedResource<LroResourceProperties> {
+  ...ResourceNameParameter<LroResource>;
+}
+
+model LroResourceProperties {
+  @doc("The description of the resource.")
+  description?: string;
+}
+
+@armResourceOperations
+interface Lro {
+  @scenario
+  @scenarioDoc("""
+    Resource PUT operation.
+    Service returns "Azure-AsyncOperation" on initial request.
+    final-state-via: Azure-AsyncOperation
+    
+    Expected verb: PUT
+    Expected path: /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Azure.ResourceManager.OperationTemplates/lroResources/lro
+    Expected query parameter: api-version=2023-12-01-preview
+    Expected request body:
+    ```json
+    {
+      "location": "eastus",
+      "properties": {
+        "description": "valid"
+      }
+    }
+    ```
+    
+    Expected status code: 201
+    Expected response header: Azure-AsyncOperation={endpoint}/subscriptions/00000000-0000-0000-0000-000000000000/locations/eastus/operations/create_or_replace
```

Comment:
SDK shall use url from `Azure-AsyncOperation` to poll, so I guess here shall be `{endpoint}/subscriptions/00000000-0000-0000-0000-000000000000/locations/eastus/lro_create/aao`?

Reply:
Thanks, this is a mistake, should be the one you mentioned.